test/unit/modules/transactions.js

```diff
@@ -329,14 +325,6 @@ describe('transactions', function () {
 				}
 			};
 
-			it('should return error for invalid schema type', function (done) {
```

Comment:
@karmacoma Removed because request schema validation is done by swagger layer and covered by functional tests.

Reply:
Ok